cmd/skuba/flags/verbose.go

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2020 SUSE LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package flags
+
+import (
+	"flag"
+
+	"github.com/spf13/pflag"
+)
+
+const (
+	verboseLevelFlagShort = "v"
+	verboseLevelFlagLong  = "verbosity"
+	verboseLevelFlagUsage = "log level [0-5]. 0 (Only Error and Warning) to 5 (Maximum detail)."
+)
+
+// GetVerboseFlagLevel returns verbose flag level.
+func GetVerboseFlagLevel() string {
+	if f := flag.CommandLine.Lookup(verboseLevelFlagShort); f != nil {
+		pflagFlag := pflag.PFlagFromGoFlag(f)
+		return pflagFlag.Value.String()
+	}
+
+	return "0"
+}
+
+// RegisterVerboseFlag register verbose flag.
+func RegisterVerboseFlag(local *pflag.FlagSet) {
+	if f := flag.CommandLine.Lookup(verboseLevelFlagShort); f != nil {
+		pflagFlag := pflag.PFlagFromGoFlag(f)
+		pflagFlag.Name = verboseLevelFlagLong
+		pflagFlag.Usage = verboseLevelFlagUsage
+		local.AddFlag(pflagFlag)
+	}
```

Comment:
should we log fatal here if this fails? I'm trying to think of why this might fail... in general I would like to see a failure here I think maybe, but might not be a big deal? :thinking:

Reply:
I can't think of a reason it would fail here.  The parsing handles most problems and will spit out the help message if something doesn't match (like "-v somestring").  Any other problem would be an internal coding error (like trying to register the flag twice) which are detected by our testing.

Reply:
Add klog back.

Now the `v` flag register by klog package. Skuba changes the name and usage output string (different to klog).

The situation _Lookup_ failed is the `v` flag is not register by klog (we replace the klog package to other logging package). For easier debugging for the engineer, I added it back.